PanModal/Presentable/PanModalPresentable.swift

```diff
@@ -11,227 +11,240 @@ import UIKit
 /**
  This is the configuration object for a view controller
  that will be presented using the PanModal transition.
-
+ 
  Usage:
  ```
  extension YourViewController: PanModalPresentable {
-    func shouldRoundTopCorners: Bool { return false }
+ func shouldRoundTopCorners: Bool { return false }
```

Comment:
Your IDE may have done this, but these whitespaces probably are deliberate and should remain 😄